tensorflow_io/oss/README.md

```diff
@@ -0,0 +1,34 @@
+# TensorFlow OSS Filesystem Extension
+
+OSS is an object storage service provided by Alibaba Cloud, see [here](https://www.alibabacloud.com/product/oss) for more information about the service.
+
+This module provides an extension that emulates a filesystem using the object storage service. The directory structures are encoded in object keys and file contents are stored in objects. The extension is implemented using [OSS C SDK](https://github.com/aliyun/aliyun-oss-c-sdk).
+
+To use the extension, first save your OSS credential in a file,  in `INI` format:
+
+```
+[OSSCredentials]
+host = cn-hangzhou.oss.aliyun-inc.com
+accessid = your_oss_access_id
+accesskey = you_oss_access_key
+```
+
+Then set environment variable `OSS_CREDENTIALS` to the path of the file.
+
+In Python code, import the extension `ossfs_op` module to use the extension with `gfile`. The files and directory URI should have `oss://` prefix, followed by a bucket name, then the directory hierarchy.
+
+```
+import tensorflow_io.oss.python.ops.ossfs_ops
+from tensorflow.python.platform import gfile
+
+gfile.MkDir('oss://your_bucket_name/test_dir')
```

Comment:
Perhaps also briefly mention how to use this with some of the existing dataset ops even though it seems pretty trivial.

Reply:
Done